### ISSUE TITLE
Widevine issue?

### ISSUE THREAD

**Issue by Queuecumber** [saiarcot895/chromium-ubuntu-build#36]
I was using the widevine component from Chrome without issue up until recently. I dont know exactly when it stopped working because I dont use Netflix too often from my computer. The only other thing I can think of that woud have caused it is that I upgraded to 18.10 a couple of days ago. Can you think of anything that might have broken recently? I tried reinstalling everything and copying over widevine from the latest chrome package.

**Comment by Queuecumber** [comment]
Just to give an update, I purged your PPA which I believe downgraded me to the Ubuntu official chromium build at 70.0.3538.67 and both netflix and the widevine test (https://bitmovin.com/demos/drm) work fine.

**Comment by saiarcot895** [comment]
Yeah, I'm seeing the issue as well. Let me see if I can figure out why it's happening.

**Comment by Queuecumber** [comment]
Just noting that it seems to still be an issue in  72.0.3590.0 which I just pulled down

**Comment by saiarcot895** [comment]
Widevine isn't related to VAAPI.

As best as I can figure out right now, Chromium just doesn't want to read the Widevine library.

**Comment by Oober** [comment]
I have also fought with this issue.

This seems to be very confusing. Firstly, I managed even to get Netflix working with the chromium-dev build. But after some trials and errors with ppapi flash, I deleted the Chromium profile and after that the Widevine did not work anymore. After purging ppa (and downgrading) the Netflix works (I had to use the UA Switcher, though), but no Widevine is listed on chrome://components.

My guess is that this would be some confusing issue related to widevine versions etc.

**Comment by saiarcot895** [comment]
Likely, or it's expecting a manifest file somewhere.

**Comment by Esokrates** [comment]
For me Widevine Content Decryption Module shows up in chrome://components but Amazon prime Video does not work, it throws me:

> Protected Content Error
To watch protected content, in the top right of Chrome, click the three dots icon, and then tap Settings > Site settings > Protected content. Under Blocked, tap Amazon.co.uk > Clear and Reset. Refresh or reload the video's detail page, and then tap Watch Now.
For further assistance, please contact Amazon Customer Service at www.amazon.de/videohelp and refer to error 7218.

In the settings protected content is allowed though.

**Comment by saiarcot895** [comment]
I took a brief look, but didn't get anywhere. Let me see if I can figure out something with the current debug-ish build I have.

**Comment by Esokrates** [comment]
@saiarcot895 Please have a look at comment 10 here: https://bugs.gentoo.org/670928#c10 
According to gentoo people this patch works: https://git.archlinux.org/svntogit/packages.git/tree/trunk/chromium-widevine.patch?h=packages/chromium

**Comment by ventz** [comment]
This should help for step by step (and extracted binary): https://blog.vpetkov.net/2019/07/12/netflix-and-spotify-on-a-raspberry-pi-4-with-latest-default-chromium/
